frontend/src/i18n/locales/de.yaml

```
@@ -280,7 +281,7 @@ manage:
       Der Seitename wird im Navigationsmenü und als Überschrift auf der Seite selber angezeigt.
       Seitennamen können jederzeit geändert werden.
     path-segment-info: >
-      Pfadsegmente dürfen nur aus Ziffern, Kleinbuchstaben und Bindestrichen (-) bestehen.
+      Pfadsegmente dürfen keine Kontroll- oder Leerzeichen enthalten.
```

I think I would even mention the special starting characters here. But it's fine for now.

---

Wie sind "Kontrollzeichen" definiert? Laut commit sollte hier dann eigentlich auch "Sonderzeichen" stehen. Ich denke, das wäre für den normalen User einfacher zu verstehen.

---

Sonderzeichen, also sowas wie `*`, `!` und `$` sind aber grundsätzlich erlaubt. Mit Kontrollzeichen meinen wir was Unicode als Kontrollzeichen bezeichnet: non-printable characters, wie `\n`, `\t` und ganz komische: https://en.wikipedia.org/wiki/C0_and_C1_control_codes

Wenn ich so drüber nachdenke, würde ich das gar nicht erwähnen, denke ich? Normale Nutzer werden niemals diese komische Art von characters eingeben. VIelleicht also:

```suggestion
      Pfadsegmente dürfen keine Leerzeichen enthalten und nicht mit <code>-+~@_!$&;:.,=*'</code> beginnen.
```

---

Hab es als PR #330 eingeworfen.